### DISCUSSION TITLE
createAsyncThunk options boilerplate

### DISCUSSION THREAD

**Discussion by zalizko** [reduxjs/redux-toolkit#1537] [Q&A]
Hello to everyone who's reading this!

I have such simple async thunk:
```javascript
export const getBookmarks = createAsyncThunk('bookmarks/get_bookmarks', async () => axios.get<IBookmark[]>(`api/bookmarks`),
  { serializeError: serializeAxiosError }
);
```

And problem is that I need to repeat options with `{ serializeError: serializeAxiosError }` in every async thunk creation using `createAsyncThunk(...)`. Could you suggest how to setup `serializeError` by default? I can wrap `createAsyncThunk` of course, but maybe there is  another solution?

**Comment by phryneas** [comment]
Your options are to either just have a `options` constant that you import everywhere and just use or to use some higher-level abstraction like RTK Query. We cannot make this globally configurable as TypeScript would not be able to pick that up.
